dashboard/app/views/layouts/_small_footer.html.haml

```diff
@@ -12,7 +12,8 @@
     # false because we're not rendering into a phone wireframe here
     phoneFooter: false,
     fullWidth: full_width,
-    channel: channel
+    channel: channel,
+    scriptYear: view_options[:script_year]
```

Comment:
Yay this is very clean!! One thing I have no answers about- how much of the things in our codebase labeled 'script' should be converted to 'unit'?

Reply:
Oof, that's a great question. I'm going to ask in the team slack channel to see what the consensus is.